### DISCUSSION TITLE
How to tell which upstream version tractusx-edc currently supports?

### DISCUSSION THREAD

**Discussion by arnoweiss** [eclipse-tractusx/tractusx-edc#719] [Q&A] [answered]
I have one specific and one general question

1. Specific: I want to know if tractusx-edc already supports the management-API endpoint `/v3/assets` as `/v2/assets` is [deprecated upstream.](https://github.com/eclipse-edc/Connector/issues/3376#issuecomment-1686010650)
2. General: If this repo included a hint from what upstream release it forks, that would allow me to check the answer to question 1 myself. If there already is one, I'd appreciate a pointer.

EDIT: Found a similar [Discussion with broken links](https://github.com/eclipse-tractusx/tractusx-edc/discussions/186). I think this should be part of the docs.

**Comment by ndr-brt** [comment, marked as answer]
The EDC version number it is specified in the gradle version catalog file:
https://github.com/eclipse-tractusx/tractusx-edc/blob/1a60ce77e4f898643f492faf21bd02e4a92613fa/gradle/libs.versions.toml#L5

There's also a way to get it programmatically, as it was implemented in this script for uploading api specs:
https://github.com/eclipse-tractusx/tractusx-edc/blob/1a60ce77e4f898643f492faf21bd02e4a92613fa/.github/workflows/publish-swaggerhub.yaml#L77
